esbuild-node-externals/README.md

```diff
@@ -9,7 +9,7 @@ When bundling with Esbuild for the backend by default it will try to bundle all
 
 ## Installation
 
-This plugin requires minimum **Node.js 12**, and **Esbuild 0.12+**.
+This plugin requires minimum **Node.js 18**, and **Esbuild 0.21+**.
```

Comment:
why did you change the minimum esbuild version here? It's didn't change in the package.json and as there is no breaking changes in these releases for our plugin it should be fine no?